packages/fhir/src/Adaptor.js

```diff
@@ -3,6 +3,15 @@ import { expandReferences } from '@openfn/language-common/util';
 
 import { handleResponse, request } from './Utils';
 
+/**
+ * Options provided to the Create request
+ * @typedef {Object} RequestOptions
+ * @property {object|string} body - body data to append to the request. JSON will be converted to a string .
+ * @property {boolean} throwOnError - Optional boolean value to throw if the error is status code 400. Default `true`
```

Comment:
Again nitty picking, but will be nice if we can add the default values in jsdoc. 
`Eg: [throwOnError=true]`. 
This will create a `default` column in a `RequestOptions` table.